docs/guides/high-availability/examples/generated/ispn-single.yaml

```diff
@@ -305,12 +311,12 @@ spec:
   expose:
     type: Route
   configMapName: "cluster-config"
-  image: quay.io/infinispan/server:15.0.11.Final
-  version: 15.0.4
+  image: quay.io/infinispan/server:15.0.15.Final
+  version: 15.0.15
   configListener:
     enabled: false
   container:
-    extraJvmOpts: '-Dorg.infinispan.openssl=false -Dinfinispan.cluster.name=ISPN  -Djgroups.xsite.fd.interval=2000 -Djgroups.xsite.fd.timeout=15000'
+    extraJvmOpts: '-Dorg.infinispan.openssl=false -Dorg.infinispan.threads.virtual=true -Dinfinispan.cluster.name=ISPN  -Djgroups.xsite.fd.interval=2000 -Djgroups.xsite.fd.timeout=15000 -Dotel.traces.sampler=parentbased_always_off'
```

Comment:
`-Dorg.infinispan.threads.virtual=true` is redundant now as we enable by default if there is sufficient parallelism. We should probably remove this from keycloak-benchmark so that it does not make it into the Keycloak docs when we copy over the files.

Reply:
I didn't realise we didn't show the `extraJvmOpts`, so this isn't really a concern.

Reply:
... and I just remembered we're deploying Infinispan separately, so it is in fact needed until https://github.com/infinispan/infinispan/pull/14579 lands in 16 :upside_down_face:

Should we be including this property in the example Infinispan CRs given this is what we're testing with?

Reply:
AFAIK it is not officially supported in RHDG, so I'd say no.